tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -101,7 +99,7 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
     os.environ["CONVERT2RHEL_ALLOW_OLDER_VERSION"] = "1"
 
     with convert2rhel(f"--no-rpm-va --debug") as c2r:
-        if "centos-6" in booted_os or "oracle-6" in booted_os:
+        if "centos-6" in SYSTEM_RELEASE or "oracle-6" in SYSTEM_RELEASE:
```

Comment:
Since we have a lot of conditions like this in our integration tests, I will leave this comment here, but if you decide to apply it, I think it would be best to apply to every other condition like that in this PR.

Reply:
Love it! Thank you for the tip @r0x0d !